nextcord/client.py

```diff
@@ -862,6 +863,23 @@ def get_sticker(self, id: int, /) -> Optional[GuildSticker]:
         """
         return self._connection.get_sticker(id)
 
+    def get_scheduled_event(self, id: int, /) -> Optional[ScheduledEvent]:
+        """Returns a scheduled event with the given ID.
+
+        .. versionadded:: 2.0
+
+        Parameters
+        -----------
+        id: :class:`int`
+            The ID to search for.
```

Comment:
"The scheduled event's ID to search for." or somethin

Reply:
all the other get_x do this, but ill follow this